src/sentry/static/sentry/app/stores/tagStore.tsx

```diff
@@ -86,6 +86,8 @@ const tagStoreConfig: Reflux.StoreDefinition & TagStoreInterface = {
           'muted',
           'assigned',
           'unassigned',
+          'triaged',
+          'untriaged',
```

Comment:
These aren't immediately surfaced:
![image](https://user-images.githubusercontent.com/549473/97192702-62b38500-1765-11eb-961e-257e5987319d.png)

until you type enough to filter out the earlier options:
![image](https://user-images.githubusercontent.com/549473/97192722-6a732980-1765-11eb-8643-f55930e94837.png)

If we use `is:` for this / other orthogonal states, I think it's important for the UI to always enumerate all the options for feature discoverability.

Nice to have: a tooltip / description that explains what each option means... maybe the other options are more self-explanatory... but I worry triaged/untriaged might not be obvious.

Reply:
Handling in #21719